src/components/Feeds/BusinessFeed.js

```diff
@@ -37,11 +37,13 @@ function BusinessFeed({
       paddingX={[null, theme.spacing.base, theme.spacing.lg]}
       width="100%"
     >
-      <BusinessFilter
-        onSearch={filters => onSearch(filters)}
-        selectedFilters={selectedFilters}
-        isSearching={loadingState === LOADING_STATE.SEARCHING}
-      />
+      <Box margin="0 auto 3rem">
```

Comment:
This was me adopting a "No outside white space" policy on this component. The internal application of the `3rem` bottom margin was making component reuse difficult.